.github/ISSUE_TEMPLATE/catch-up-issue.md

```
@@ -0,0 +1,15 @@
+about: A "catch-up"-Issue is used to track missing documentation and ensure that it
+  is caught up.
+title: ''
+labels: catch-up
+assignees: ''
+
+---
+
+# Summary
+<!-- please describe, what this documentation issue will be about -->
+
+# Reference
```

Markdownlint (0.35.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)